packages/logger/.gitignore

```diff
@@ -0,0 +1,25 @@
+.DS_Store
+node_modules
+/dist
+/types
+
+/tests/e2e/reports/
+selenium-debug.log
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw*
```

Comment:
Does it makes sense to have a `.gitignore` file per package? We already have one at `root` level. Just thinking out loud here.

Reply:
I thought the same, but as each package has its own dependencies and output folder I would leave it there for the moment. In other words, see this `.gitignore` strictly related with the dependencies and the build configuration of each package, which goes in the `package.json`